crates/viewer/re_data_ui/src/instance_path.rs

```diff
@@ -59,106 +70,222 @@ impl DataUi for InstancePath {
                     if indicator_count > 1 { "s" } else { "" }
                 ),
             );
```

Comment:
since the `return;` got removed here we're still evaluating the image preview if this is a single line. I don't know what this could break in the ui (haven't spotted anything) but this might be wrong? (could be that something down the line opts out, but I'm not seeing it in this diff at least)

Reply:
The idea is that if we put an `Entity` in a list view, we should see a small preview image, like we currently do for `ImageEncoded` blobs: 

<img width="268" alt="image" src="https://github.com/user-attachments/assets/11b91251-c5ee-40a6-9298-c758b7cbe671">

Reply:
I prefer this way of writing it to early returns anyway